src/patterns/durations/index.njk

```diff
@@ -38,14 +38,35 @@ To help users enter a valid duration, you should:
     })
 }}
 
-### Errors
+### Error messages
+
+Use the [correct errors pattern](/patterns/error-validation) and show the error details above the duration fields.
+
+{{
+    patternlibExample({"path": "components/duration/examples/duration-address-error/index.njk"})
+}}
```

Comment:
This example is incorrect now that you've done the work to highlight a single field.  So I would change this one to a simple all empty "Enter a duration" error.